package.json

```diff
@@ -52,6 +52,7 @@
         "rollup-plugin-visualizer": "^5.12.0",
         "sass": "^1.79.4",
         "serve": "^14.2.3",
-        "tslib": "^2.7.0"
+        "tslib": "^2.7.0",
+        "typescript": "^5.6.3"
```

Comment:
Why add the `typescript` dependency in this PR?

Reply:
If `typescript` is not globally installed, the developer will encounter the error: "Error: Cannot find package 'typescript' imported from..."

Reply:
I think it's OK to leave it. 😄